solverdummy/solverdummy.py

```diff
@@ -31,24 +38,41 @@
 
 dimensions = interface.get_dimensions()
 vertices = np.zeros((n, dimensions))
+readData = np.zeros((n, dimensions))
+writeData = np.zeros((n, dimensions))
+
+for x in range(0,n):
+    for y in range(0,dimensions):
+        vertices[x,y] = x
+        readData[x,y] = x 
+        writeData[x,y] = x
 
 vertex_ids = interface.set_mesh_vertices(mesh_id, vertices)
```

Comment:
It looks like all vertices are still zero, correct?

Reply:
No. The vertex coordinates are set as `vertices[x,y] = x`. We get
identical values for the x and y coordinate of each vertex. For `n = 3`
we get `vertices = [[0,0],[1,1],[2,2]]`.

Note: I would suggest to rename `y` to `d` for dimensions. The `y` is
misleading. Similarly for `x`. Better use something like `i` or `idx`
for vertex index. Alternatively, one might just use random numbers for
the vertex coordinates, since I think it does not really matter. Or does
it, since we are now reading & writing data?

Another note: Instead of `vertices[i, d] = x` you could also write
`vertices[i, :] = [x, y]` to save the inner loop (same for `readData`
and `writeData`). Of course, you will have to set `x, y` before. This
could also be done via `x, y = i, i` (or whatever values you want to use).